static/js/app.js

```diff
@@ -244,10 +244,37 @@ function getSelectedLicense() {
   free = document.getElementById('free-license');
   nonCommercial = document.getElementById('non-commercial-license');
   commercial = document.getElementById('commercial-license');
+  noLicenseResults = document.getElementById('no-license-results');
 
   // Add on change event listener to the license type dropdown.
   licenseType.addEventListener('change', (e) => {
     licenseTypeValue = e.target.value;
+    
+  // Add "+" before non-commercial to make the term required when searching lunr.
```

Comment:
Adding `+` came from the lunr docs as a way to improve the results when there are multiple terms in the search value.